freqtrade/rpc/telegram.py

```diff
@@ -652,9 +694,9 @@ async def _status_msg(self, update: Update, context: CallbackContext) -> None:
                         "*Open Order:* `{open_orders}`"
                         + ("- `{exit_order_status}`" if r['exit_order_status'] else ""))
 
-            lines_detail = self._prepare_order_details(
-                r['orders'], r['quote_currency'], r['is_open'])
-            lines.extend(lines_detail if lines_detail else "")
+            # lines_detail = self._prepare_order_details(
+            #     r['orders'], r['quote_currency'], r['is_open'])
+            # lines.extend(lines_detail if lines_detail else "")
```

Comment:
is this PR still [wip] ?? looks pretty much complete if i'm honest ...

aside from that, we would probably want to remove these lines (instead of commenting them) - obviously also applies to the test file ...

Reply:
Forgot to remove the WIP tag :D. Gonna remove the comments